plugin/storage/integration/badgerstore_test.go

```diff
@@ -52,7 +53,8 @@ func (s *BadgerIntegrationStorage) initialize(t *testing.T) {
 }
 
 func (s *BadgerIntegrationStorage) cleanUp(t *testing.T) {
-	s.factory.Purge()
+	ctx := context.Background()
+	s.factory.Purge(ctx)
```

Comment:
try to avoid storing context
```suggestion
	s.factory.Purge(context.Background())
```

Reply:
also. add require.NoError()